Kitodo/src/main/webapp/pages/processes.xhtml

```diff
@@ -14,35 +14,35 @@
 <ui:composition
         template="/WEB-INF/templates/base.xhtml"
         xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
-        xmlns:f="http://xmlns.jcp.org/jsf/core"
         xmlns:h="http://xmlns.jcp.org/jsf/html"
         xmlns:p="http://primefaces.org/ui">
     <ui:define name="content">
         <h:outputScript name="js/filter.js" target="body" />
         <p:panel id="listWrapper">
             <p:panel styleClass="content-header">
                 <h3>#{msgs.prozessverwaltung}</h3>
-                <h:form>
+                <h:form id="addForm">
                     <p:menuButton value="#{msgs.newElement}" icon="fa fa-plus-circle fa-lg" iconPos="right" styleClass="button-filled-green">
-                        <p:menuitem value="#{msgs.newProcess}" outcome="/pages/ProzessverwaltungBearbeiten" icon="ui-icon-plusthick" />
+                        <p:menuitem value="#{msgs.newProcess}" outcome="/pages/processEdit?id=0" icon="ui-icon-plusthick" />
```

Comment:
Is this hard coded parameter `id` with value `0` really what you want? This is maybe related to my comment above to this maybe wrong null check.